static/src/javascripts/projects/commercial/modules/hosted/gallery.spec.js

```diff
@@ -0,0 +1,158 @@
+// @flow
+import interactionTracking from 'common/modules/analytics/interaction-tracking';
+import HostedGallery from './gallery';
+import { galleryHtml } from './gallery-html';
+
+jest.mock('lib/detect', () => ({
+    hasPushStateSupport: jest.fn(),
+    getBreakpoint: jest.fn(),
+    hasTouchScreen: jest.fn(),
+}));
+jest.mock('common/modules/analytics/interaction-tracking', () => ({
+    trackNonClickInteraction: jest.fn(() => Promise.resolve()),
+}));
+jest.mock('lib/load-css-promise', () => ({
+    loadCssPromise: Promise.resolve(),
+}));
+jest.mock('commercial/modules/dfp/performance-logging', () => ({
+    moduleStart: jest.fn(),
+    moduleEnd: jest.fn(),
+}));
+
+let gallery: any;
+const noop = () => {};
```

Comment:
[There's a lib for that](https://github.com/guardian/frontend/blob/master/static/src/javascripts/lib/noop.js) 😄